src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
+  const UNIQUE_KEYS = [ ...new Set(KEYS) ];
```

Comment:
The variable name 'UNIQUE_KEYS' is not descriptive enough. It's better to use a name that reflects the purpose of the variable. For example, 'uniqueRobotValues' would be a more appropriate name.